src/frontend/shared/app-layout.vue

```diff
@@ -72,6 +74,16 @@ const setDimensions = () => {
   }
 };
 
+watch(
```

Comment:
How about setting a simple key name. It is scoped inside the app domain so it won't clash with other open browser tabs:

```suggestion
const sidebarKey = 'sidebar-state';

watch(
```